client/components/Help/Help.js

```diff
@@ -44,13 +82,15 @@ class Help extends Component {
                 </div>
                 <ReactCSSTransitionGroup
                     className="problems-container"
-                    transitionName="new-problem"
+                    transitionName="problems-container"
                     transitionEnterTimeout={500}
                     transitionLeaveTimeout={500}>
                     {isNewProblemVisible ? <NewProblem addButtonClick={this.addButtonClick}/> : null}
                     <div className="problems">
                         {problems}
                     </div>
+                    {isAnswersListVisible ?
+                        <AnswersList currentProblem={currentProblem} hideButtonClick={this.hideButtonClick}/> : null}
```

Comment:
why you created that reducer currentProblem??
you already have problems reducer. pass problem from that reducer. DRY.